apps/slippistats/slippistats.star

```diff
@@ -0,0 +1,277 @@
+    "26_2": "UklGRlYBAABXRUJQVlA4TEoBAAAvF8AFEH9gJAASJlCZQLU53AteIHrBOUYCIOEGUxzsFSNRItUJRiORpJx8+xVwH54a1EAiqTD/AYCVIET45YOTTSJG9u1BWOxRU9otOoBrbf/a5MOy4zbln3O6N7i7bNzCPz2cH05GbsB9c4eJrrhm69Qdd91c6r8h5R4i+j8BsBMRIbpUhUgyojmEJrQkxSYT9RxNF4aNhHOtOEckKQAceo5Td68lGzaR0+tz53y5AKSJ4sgg85ILkGmq+23ilWdMgNLUNV+Er7gb8qApaS3wFjl5a4Ocq9Wcsz6+WmqDnOzIyXMPvvFiB0B7Int1+O2H2wG5p1r3674fqxkgyxh6GN+wBhVIqsgZ2Q0uNY4o8re+KGeyxzezM6OQuc051cHavYaDBI9JRPTLvEPwhJiZzYqNPwUer9cbDvvy99tgu/T5xkyLAEn/AgA=",
+    "26_3": "UklGRlYBAABXRUJQVlA4TEkBAAAvF8AFEH9gpo2UG58NC28Hj1gCGW/JSCQpSCJ8BiJQg97Ykx8BizQSScrJl+CoQQ0s/YcKyJfzHwQ6lTYJLlilSLEFLIE7/nHG75PbzAIUW9vWOG9c5f9nCmoY/OTPCpIyo6xjZgYdNSCzhE6XUM3Mjhn+86UkuZnuIaL/EwA355wjO9OECKvZ4kIXelhxsZBZqJtCdXFhNBYVirACIG4WGmZTY0R1icIV2VR47QEQFEXOJtGpB2BBrelr+4NOfAAPao3SoQz5wdrmwo03X87ClxfseFw3jiRlTr1ge/mFpU2bX3SWD/BeEWtY+rqnPID1an1vzfK+rwDgferW+05L36YCxqsLl3tuT2eWFXYxWFW4tyhPuk8UnuigwuHb8d7ptpx0wrIs656omyP9QESUqG55UpC2bfvxUZa2euGal3I90ccBxv8FAAA=",
+    "26_4": "UklGRlQBAABXRUJQVlA4TEgBAAAvF8AFEH9gpo2UG1/Cvw10sqIPCSDHaMAkvEEk+pJHW4mLvmGmto1X4W9mUUQgn2oy2BjnPxBYYqDhuDgEc6wHe685aqmBmqaNF0D5/3+NuqRvZBGHuD+elxa3F+5aSe7u9hT+O/SHS7sX1bXfkXjRbAl3l5Nf2BHR/wmAk4gIoaUqRIQVKk0kisQIxSHDzfREU1gOEkZ5RrqIUACkmemGWVEeZTlE+qavIv3DBSBeZAR2mZ9dgIxXK+zDU35aBCheLfcF2M3zkDWjEeWfdmDNXoC86k00Ln3sfl6AvJ1Mz67YtflxEqAWEVO6bv/wBCBb1NbjSt9P6xRArdbByX5V664CSUXpG41fz4MbivztL0y/PfI9NT0ptFjP6R1fvS1DNWH6IhHRL3MTQX9nZl4sqvIr0DVN83h8OdULcKx6vTuLrQRI+hcA",
+}
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "code",
+                name = "Slippi Code",
+                desc = "Ex: TRB#328",
+                icon = "user",
+            ),
+        ],
+    )
+
+def main(config):
+    totalTimeCached = cache.get("cache_totalTime")
+
+    if totalTimeCached != None:
+        print("Hit! Displaying cached data.")
+        totalGames = cache.get("cache_totalGames")
+        winrate = cache.get("cache_winrate")
+        iconName = cache.get("cache_iconName")
+
+        totalTime = totalTimeCached
+
+    else:
+        print("Miss! Calling SLP API.")
+        code = config.str("code", DEFAULT_CODE)
+        code = code.replace("#", "-")
+        GET_URL = CHART_SLP_SERVER_URL + code
+
+        rep = http.get(GET_URL)
+        if rep.status_code != 200:
+            fail("Chart SLP Request Failed with code")
+        totalTime = rep.json()["totalTime"]
+        totalTime = totalTime.split(":")
+        totalTime = humanize.comma(totalTime[0]) + " hrs"
```

Comment:
totalTime needs to be converted to an int here

Reply:
Good call. Addressed.